docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -280,6 +280,24 @@ Additionally, the `{{internal.jwt}}` template variable will be replaced with
 a JWT token signed by Teleport that contains user identity information. See
 [Integrating with JWTs](../jwt/introduction.mdx) for more details.
 
+The JWT token can be optionally configured to exclude traits/roles:
```

Comment:
Can this be put in a separate guide section e.g. `## Configuring JWT token` or something like this. This way it will be easy to link to if someone's running into the large header issue.